components/card.css

```diff
@@ -0,0 +1,256 @@
+@import url(/main.css);
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@1,300&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Raleway:ital,wght@1,300&family=Source+Code+Pro:ital,wght@1,200&display=swap');
+
+*{
+padding: 0;
+    margin: 0;
+}
+
+.megaContainer{
+    margin-top: 2rem;
+    display: flex;
+   gap: 2rem;
+   align-items: center;
+   flex-wrap: wrap;
+   padding-left: 5rem;
+ 
+}
+
+.card-skeleton{
+    width: 15.625rem;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    /* margin: 5rem 0; */
+    background-color: white;
+   height: min-content;
+
+}
+
+.image{
+    width: 100%;
+}
+
+.footer-button{
+    display: flex;
+    justify-content: space-around;
+    align-content: center;
+    padding-top: 1rem;
+
+}
+
+.card-button{
+    width: 100%;
+   background-color: var(--primary-color);
+    padding: 1rem;
+    border: transparent;
+}
+
+.content{
+    padding-left: 1rem;
+
+    /* font-family: 'Raleway', sans-serif; */
```

Comment:
Don't add unnecessary comments and also format the code (Prettier , Beautify)

Reply:
fixed , can you check